sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -255,61 +244,58 @@ export class EventHubConsumerClient {
    * other subscribers.
    *
    * @param consumerGroupName The name of the consumer group from which you want to process events.
-   * @param onReceivedEvents Called when new events are received.
    * @param partitionId A partition id to subscribe to.
    * @param options Options to handle additional events related to partitions (errors,
    *                opening, closing) as well as batch sizing.
    */
 
   subscribe(
     consumerGroupName: string,
-    onReceivedEvents: OnReceivedEvents,
     partitionId: string,
-    options?: SubscriptionOptions
+    options: SubscriptionOptions
   ): Subscription; // #2
   /**
    * Subscribes to multiple partitions.
    *
    * Use this overload if you want to coordinate with other subscribers using a `PartitionManager`
    *
    * @param consumerGroupName The name of the consumer group from which you want to process events.
-   * @param onReceivedEvents Called when new events are received.
    *                         This is also a good place to update checkpoints as appropriate.
```

Comment:
I think this line also needs to be removed.

Reply:
It is removed :)